signac/core/jsoncollection.py

```diff
@@ -0,0 +1,138 @@
+# Copyright (c) 2020 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+import os
+import json
+import errno
+import uuid
+
+from .collection_api import SyncedCollection
```

Comment:
I find the name of that module weird. If it only contains the abstract base classes, maybe we should call it `synced_collection_abc`? @vyasr Do you have a better idea?

Reply:
I think that module should be called `synced_collection` and only define the parent class for the hierarchy (`SyncedCollection`). The original name is a holdover from my original prototype where everything was in one file.

Reply:
This has been completed. Marking resolved.